src/Parallel/MemoryMonitor/Tags.hpp

```diff
@@ -0,0 +1,39 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <string>
+#include <unordered_map>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "Options/Options.hpp"
+#include "Utilities/PrettyType.hpp"
+
+namespace mem_monitor {
+/*!
+ * Gives full subfile path to the dat file for the memory monitor of a
+ * parallel component
+ */
+template <typename ParallelComponent>
+std::string subfile_name() {
+  return "/MemoryMonitors/" + pretty_type::name<ParallelComponent>();
+}
+
+namespace Tags {
+/*!
+ * \brief Tag to hold memory usage of parallel components before it is written
+ * to disk.
+ *
+ * \details The types in the unordered_map are as follows:
+ *
```

Comment:
Do you want to make this a code block using triple ticks on the line before and after? Up to you :)